src/parser.rs

```diff
@@ -245,53 +246,53 @@ pub(crate) rule table() -> Table
     }
 
 pub(crate) rule step() -> Step
-    = comment()* pa:position!() k:keyword((env.keywords().given)) __ n:not_nl() pb:position!() _ nl_eof() _
+    = comment()* pa:position!() k:keyword((env.keywords().given)) _ n:not_nl()? pb:position!() _ nl_eof() _
```

Comment:
I don't think this is a way to do it. Name of the `Step` is matched over in regex and allowing it to be empty is really confusing.

Reply:
Reverted back (except requiring whitespace after the keyword since it now is part of the keyword).